Lib/glyphsLib/parser.py

```diff
@@ -135,17 +134,9 @@ def _parse(self, text, i, _parsing_unicodes=False):
         m = self.hex_re.match(text, i)
         if m:
             parsed, value = m.group(0), m.group(1)
-            decoded = binascii.unhexlify(value)
+            decoded = bytearray(binascii.unhexlify(value))
```

Comment:
I thought about returning something else than mere `bytes`, e.g. a bytearray object, so that in the Writer we can detect that, and serialize it as a unquoted, hexadecimal string within `<` and `>` (see below). If we use generic bytes, then it's tricky to distinguish between python2 `str` values containing ascii text, which the user may have set in a userData dict, and actual binary data which is meant to be hexlified. 
I know, bytearrays are mutable. If that's a problem, we may wish to subclass the built-in `bytes`. But bytearray is handy because it's built-in. And it's unequivocally binary data.